.travis.yml

```diff
@@ -0,0 +1,28 @@
+language: python
+python: 2.7
```

Comment:
@jensens is the other way around: by saying `sudo:  false` you tell Travis to use the container-based infrastructure; if you dont say anything then `sudo:  true`:

http://docs.travis-ci.com/user/migrating-from-legacy/

any other comment? this is working...